internal/contour/route.go

```diff
@@ -283,6 +268,27 @@ func actionroute(r *dag.Route, services []*dag.Service) *route.Route_Route {
 	return &rr
 }
 
+func clusterrouteaction(services []*dag.Service) route.Route_Route {
```

Comment:
Websockets will still be broken if a user attempts to create an IngressRoute with multiple service backends & websockets enabled, correct?

Reply:
That is true, but at the moment they're broken for all use cases. I agree we should give an error and for the multi backend case, but I want to keep this PR easy to backport.